tests/beacon/types/test_states.py

```diff
@@ -65,15 +65,17 @@ def test_defaults(sample_beacon_state_params):
 def test_num_validators(expected,
                         max_deposit,
                         empty_beacon_state):
-    validator_registry = [
-        mock_validator_record(
-            pubkey,
-            max_deposit,
-        )
-        for pubkey in range(expected)
-    ]
     state = empty_beacon_state.copy(
-        validator_registry=validator_registry,
+        validator_registry=[
+            mock_validator_record(
+                pubkey,
+            )
+            for pubkey in range(expected)
+        ],
+        validator_balances=(
+            max_deposit
+            for _ in range(10)
```

Comment:
Though it might not matter, should this be `expected`? The parameters in this test look strange to me, why we are using `[(0), (1)]` not `[0, 1]` if we are using the variable `expected` as integers.

Reply:
`[(0), (1)]` means testing twice, not a list/tuple of integers.